apps/api/src/app/feeds/usecases/delete-feed/delete-feed.usecase.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { FeedRepository, FeedEntity, DalException } from '@novu/dal';
+import { DeleteFeedCommand } from './delete-feed.command';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+
+@Injectable()
+export class DeleteFeed {
+  constructor(private feedRepository: FeedRepository) {}
+
+  async execute(command: DeleteFeedCommand): Promise<FeedEntity[]> {
```

Comment:
I see the E2E test is missing for this on the API. Would be great if we can have it before merging :)